contentcuration/contentcuration/frontend/shared/views/channel/ChannelThumbnail.vue

```diff
@@ -121,11 +121,11 @@
   import ActionLink from '../ActionLink';
   import Uploader from 'shared/views/files/Uploader';
   import FileStatusCard from 'shared/views/files/FileStatusCard';
-  import FileStatusText from 'shared/views/files/FileStatusText';
+  import FileStatusText from 'frontend/channelEdit/views/files/FileStatusText';
```

Comment:
Seems a bit unusual for something inside `shared` to reference something outside of `shared`?

Reply:
There is currently work being done on the file components, so I didn't want to move the files to avoid the potential merge conflicts. Once that work is done though, will be moving to the shared folder